src/umpire/event/event.hpp

```diff
@@ -15,10 +15,45 @@
 #include <utility>
 
 #include "camp/camp.hpp"
+#include "camp/resource.hpp"
 #include "umpire/event/recorder_factory.hpp"
 #include "umpire/util/Macros.hpp"
 
+namespace camp {
+namespace resources {
+inline namespace v1 {
+
+inline std::string to_string(const camp::resources::Resource& r)
```

Comment:
How about putting this in camp instead?

Reply:
I agree with this, but shouldn't hold up this PR.